src/Common/Serialization/HtmlSanitizer.cs

```diff
@@ -0,0 +1,22 @@
+using System.Text;
+using Ganss.Xss;
+
+namespace Passwordless.Common.Serialization;
+
+public static class HtmlSanitizer
+{
+    private static readonly Ganss.Xss.HtmlSanitizer _htmlSanitizer = new(new HtmlSanitizerOptions
```

Comment:
this should be `HtmlSanitizer`

Reply:
@jrmccannon Could you clarify your comment? I'm not following.

Reply:
Naming conventions from .editorconfig says it should be `HtmlSanitizer` instead of `_htmlSanitizer`

Reply:
You cannot name it `HtmlSanitizer` because it collides with the class' name, I've just renamed it to `Instance`.

Reply:
OK to me